exec/jdbc/src/test/java/org/apache/drill/jdbc/test/TestJdbcQuery.java

```diff
@@ -403,4 +407,147 @@ public void testConvertFromInEmptyInputSql() throws Exception {
         .sql("SELECT CONVERT_FROM(columns[1], 'JSON') as col1 from cp.`empty.csv`")
         .returns("");
   }
+
+  @Test
+  public void testResultSetIsNotReturnedSet() throws Exception {
+    Statement s = null;
+    try (Connection conn = connect()) {
```

Comment:
Please rewrite tests, since looks like a connection is closed before closing the statement [1].

[1] https://docs.oracle.com/javase/specs/jls/se8/html/jls-14.html#jls-14.20.3.2